### PR TITLE
Kp mysamples 852

### PR DESCRIPTION
**Pull request by ramavavi**
This PR addresses the issue: https://github.com/episphere/connect/issues/852

**Documents for reference:** 

KPCO: https://nih.app.box.com/file/790077248266?s=p0o37qk0ao4zou37korimmxdq6ki06gb
KPGA: https://app.box.com/s/9bkqifdw5zz6kleaj6j12167d0hscofc
KPHI: https://nih.app.box.com/file/790077372511?s=tmxtgx1hh461fwgra3jerbts3bbd3ug7
KPNW: https://nih.app.box.com/file/790079045588?s=k9xap14sgt11wq184dwpl03clkgykog7

Changes made in this PR are:

1. Added more descriptive text around home collection kits, removed COVID-19 information,  removed “app” after MyConnect.
2. Renamed the subheading to "How Do I Donate My Mouthwash Sample?"
3. Made some changes to the text under some of the sub-heading.

### CODE REVIEW

**Review by jhflorey — APPROVED** [review]

File: js/pages/samples.js [hunk 1]
```diff
@@ -234,12 +234,12 @@ export const renderSamplesPage = async () => {
                     <div class="row" style="width:100%">
                         <div class="consentHeadersFont" style="color:#606060;width:100%">
                             <div>
-                                How Do I Donate My Saliva Sample?
+                                How Do I Donate My MouthwashSample?
```
Comment (Gbarra9) [inline js/pages/samples.js:237]: Should there be a space between Mouthwash and Sample?

**Review by Gbarra9 — APPROVED** [review]
Looks good

File: js/pages/samples.js (hunk 1 above)
Comment (ramavavi) [reply js/pages/samples.js:237]: No Gene, there no space needed.

Comment (we-ai) [reply js/pages/samples.js:237]: > Should there be a space between Mouthwash and Sample?

I believe so.

**Review by we-ai — COMMENTED** [review]
Looks good. Please answer the question Gene asked.

File: js/pages/samples.js (hunk 1 above)
Comment (ramavavi) [reply js/pages/samples.js:237]: Yes, a space is required between Mouthwash and sample. Adding the space, thank you.

**Review by we-ai — APPROVED** [review]

**Pull request merged**